MailerQ/Core/QueueManager.cs

```diff
@@ -30,6 +30,29 @@ public QueueManager(MailerQConfiguration configuration)
 
         public QueueManager(IOptions<MailerQConfiguration> options) : this(options.Value) { }
 
+        public void DeclareDeadLetterQueueForPublish(
+            string name,
+            bool addQueueNamePrefix = true,
```

Comment:
Is delegating the resolution of the prefix to the library a good idea? or should why simply use the full name?

Reply:
The library follow the principle _convention over configuration_ as default follow convention, and allow modification with configuration. In this case the convention is add the _QueueNamePrefix_, and allow not add it changing the value of the parameter

Call the method only with the name of the queue, will follow the convention.
This is the same that append when subscriber methods without indicate a queue name, will use the _QueueNamePrefix_ with the default queue name of _MailerQ_. But allow set a custom queue name.